src/malli/transform.cljc

```diff
@@ -89,15 +89,15 @@
       :else x)
     x))
 
+(def ^:private uuid-re
+  #"^[0-9a-f]{8}-[0-9a-f]{4}-[1-8][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$")
```

Comment:
Hello, thanks a lot for this helpful commit! I have noticed that the nil UUID (made of 0) is no longer matched after this regex-update, is it on purpose? I suggest we update this regex again so as to match the nil UUID.